t/control/schema.t

```diff
@@ -90,7 +90,7 @@ __DATA__
                     }
                 }]]
                 )
-            ngx.satus = code
+            ngx.status = code
```

Comment:
Hi @spacewander, please take a look at this line

Reply:
it is right one